.github/workflows/go.yml

```diff
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go: ['1.14', '1.15', '1.16']
+        os: ['ubuntu-20.04']
+
+    name: ${{ matrix.os }} / Go ${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: make all
+
+    # GitHub Actions' VMs don't have KVM. Firecracker itself won't work.
```

Comment:
if we are going to use github actions, I would much prefer to host our own github action instance with kvm to run the tests

Reply:
Yeah. GitHub Actions could potentially replace BuildKite altogether. But for now, I'd like to have some here to make sure firectl works on all Go versions.